src/main/java/mate/academy/dao/CinemaHallDao.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.dao;
+
+import java.util.List;
+import mate.academy.model.CinemaHall;
+
+public interface CinemaHallDao {
+    CinemaHall add(CinemaHall cinemaHall);
+
+    CinemaHall get(Long id);
```

Comment:
from common mistakes
Let’s return Optional in get() method on dao layer, and the entity on service layer